cypress/utils/commands.js

```diff
@@ -383,3 +406,18 @@ Cypress.Commands.add('setupTestData', (endpoint, mappingFiles, dataFiles) => {
 
   return chain;
 });
+
+Cypress.Commands.add('login', () => {
+  // much faster than log in through UI
+  cy.request({
+    method: 'POST',
+    url: '/auth/login',
+    body: {
+      username: Cypress.env('username'),
+      password: Cypress.env('password'),
+    },
+    headers: {
+      'osd-xsrf': true,
+    },
+  });
+});
```

Comment:
lets add this change once we have SECURITY_ENABLED tests

Reply:
`changeDefaultTenant` is meaningful only when security is enabled.